### ISSUE TITLE
[Firebase Analytics] The parameter value in GA 360 for logEvent displays the "error_value" when the parameter value is less than 500 characters.

### ISSUE THREAD

**Issue by chunglv91** [firebase/firebase-android-sdk#5778]
* Android Studio version:  Flamingo | 2022.2.1
  * Firebase Component: Firebase Analytics
  * Component version: `firebase-bom:32.7.4`

#### Steps to reproduce:

When I send a log event with a parameter value length(< 500 characters), it displays the `error_value: app_deeplink_url` message.

#### Relevant Code:

```
firebaseAnalytics = FirebaseAnalytics.getInstance(this)
firebaseAnalytics.logEvent(
    "app_deeplink_triggered", bundleOf(
    "app_deeplink_url" to "Path > 100 and path < 500 characters"
))
```

**Comment by chunglv91** [comment]
@lehcar09 
Sorry for the confusion. Could you please help me by trying with any string that has a length greater than 100 characters? Thanks!!

**Comment by lehcar09** [comment]
Apologies for the silence here.  Per checking, there’s already an existing report about this issue and we're already working on it. This is internally being tracked in b/327589091.

**Comment by abenavent96** [comment]
Any update of this issue?

**Comment by lehcar09** [comment]
Hi folks! The issue has been fixed and was rolled with BOM version 33.1.0. You can check the release notes from [here](https://firebase.google.com/support/release-notes/android#analytics_v22-0-1).

If the issue persists, please ensure that the device has Google Play Services version [24.02.xx](http://24.02.xx/).

That said, I’ll be closing this thread now. Let me know if we need to reopen the issue for further investigation.
